cohttp-eio/src/cohttp_eio.mli

```diff
@@ -90,3 +92,94 @@ module Server : sig
 
   val not_found_handler : handler
 end
+
+module Client : sig
+  type response = Http.Response.t * Eio.Buf_read.t
```

Comment:
To check the response headers or the status with this client I should do something like
```ocaml
ler res = Client.get "whatever" in
let status = Http.Response.status (fst res) in 
...
```
right? Can we add it to the example or the documentation?

Reply:
Ack. I will add it to the tests.

Reply:
Added docker_client.ml which demonstrates the usage.